### DISCUSSION TITLE
Rename Default skin

### DISCUSSION THREAD

**Discussion by webdevs-pro** [elementor/elementor#15546] [Developers Community]
I`m adding skins to my custom widget and want to rename "Default" skin name
![1623692582](https://user-images.githubusercontent.com/14045233/121936150-fa480100-cd51-11eb-9d08-191b3713713d.png)

**Comment by Aviad-Herman** [comment]
Hey,
Skins is not part of Elementor. I think you are using an external plugin.

**Comment by webdevs-pro** [comment]
Hold on a minute, skins API is a part of Elementor API. And I want to rename Default skin because I am developing custom Elementor widgets and some widgets will use this skins API like Elementor Pro posts widget. It works great but I need to rename Default widget skin which created automatically. @Aviad-Herman why did you close this issue?

**Comment by Aviad-Herman** [comment]
Sorry, misunderstanding

**Comment by webdevs-pro** [comment]
If someone else looking for this just add `protected $_has_template_content = false;` at the beginning of your widget class, this will disable default skin. Default skin name is harcoded.

**Comment by Aviad-Herman** [comment]
Great solution meanwhile.
Thanks for sharing @webdevs-pro .

I will turn this thread to a GitHub discussion so more people can see it. Thanks!
